utils/ifupdown.sh.in

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
i'd prefer to keep `/bin/sh` ; because it can be symlinked to `/bin/bash or other shells
some distros use other shells as defaults